### ISSUE TITLE
Image Classification Template changes for .NET Conf

### ISSUE THREAD

**Issue by rustd** [dotnet/machinelearning-modelbuilder#235]
- [x] Scenario screen - Update description as below
Classify images into 2+ categories (image-classification) e.g. predict type of flowers.

- [x] DataPreview should have the total count of images across all the subfolders eg.
Total images 1000. Showing 12/20.

- [x] Train progress screen add one of the following the following
Processing images : 1/40 or 
Neural Network Architecture: ResNet (exact version used for training)

- [x] Evaluate screen Output section summary
**ML Task - image-classification
Dataset - PathToTopLevelForolderImages
Output to predict (Labels) - TopLevelFolderName
Input columns (Features): Don't show this row
Neural Network Architecture - ResNet**  (exact version used for training)
Best Model - Same
Best Model Accuracy - Same
Training time - Same
Models Explored (Total) - Same

- [x] Accuracy on Train progress and Evaluate screen
Accuracy cannot be 100%. Can we max out at 95? or always subtract 5 from the actual accuracy

- [x] Code screen
Generated code should work in the user's app when they call the ConsumeModel.Predict().

- Today it is giving errors saying Microsoft.ML/ Microsoft.ML.ImageAnalytics/ Microsoft.ML.OnnxTransformer dlls not found when you launch the app.
- Generated code should work for all images sizes and shouldn't require featureization during the predict call.
Generated code should be able to take in all the images and featureize it (resize etc) so it can used for prediction

- [x] Generated Code
Generated code does not use the DNN featurizer in the training pipeline
![image](https://user-images.githubusercontent.com/1132652/65069877-6eb4fc80-d940-11e9-97aa-779570b0b812.png)

- [x] Datasets

- http://download.tensorflow.org/example_images/flower_photos.tgz
- https://www.kaggle.com/paultimothymooney/chest-xray-pneumonia
- https://www.kaggle.com/iarunava/cell-images-for-detecting-malaria
- https://www.kaggle.com/paultimothymooney/blood-cells

**Comment by beccamc** [comment]
@rustd You just want one top level folder name for the "Output to predict (Labels)". Like this?

![image](https://user-images.githubusercontent.com/9122518/65069944-9310d900-d940-11e9-8e20-fa7f86631be1.png)

**Comment by rustd** [comment]
Label instead of Labels. Apologies I had it incorrect earlier

**Comment by beccamc** [comment]
@rustd I've a [pr](https://github.com/dotnet/machinelearning-tools/pull/329) up with the first 4 tasks. 

On tweaking the accuracy to be lower. I changed the reported accuracy in the summary to max at 99%.  I think you're saying to also change the accuracy shown in the output?
![image](https://user-images.githubusercontent.com/9122518/65088013-ae91d900-d96c-11e9-9ff9-7e643ce41e68.png)

I'm thinking it may be odd to customers to have this reported inaccurately. On one hand, it doesn't hurt to have it be more accurate than what they expected. On the other, it would be confusing to see a report of 100 in one spot, and 95 in another. I'm also concerned that when they start to mess with the underlying models themselves, they'll think they did something wrong with a 100% accuracy. 

What do you think about showing a data bubble instead on 100% accuracy, explaining that its caused by a small training set that has limited data for testing?

**Comment by rustd** [comment]
Thank you. The accuracy tweak is jut for the demo and we won't ship it to customers. We don't have to change the accuracy in the Output window since we won't demo it.

Long term we should investigate on why are we getting a 100% accuracy. We just don't know how small is a small dataset. We do have a learn more link on understanding model performance and in the tutorial we could list our all cases of getting a 100% accuracy.

**Comment by beccamc** [comment]
Sounds good @rustd!  I can easily update that accuracy value. I'm waiting to see which data set will be picked for the demo. Depending on what it is, we may not need to change the accuracy? 

Jake had a description on why we're getting 100%. I'll get it and add the details here.

**Comment by beccamc** [comment]
Closing this as the .Net Conf is over. We have new work items to capture anything remaining #254, #248
